x/auth/ante/ante_test.go

```diff
@@ -543,83 +532,40 @@ func TestAnteHandlerSetPubKey(t *testing.T) {
 	require.Nil(t, acc2.GetPubKey())
 }
 
-func TestProcessPubKey(t *testing.T) {
-	app, ctx := createTestApp(true)
-
-	// keys
-	_, _, addr1 := types.KeyTestPubAddr()
-	priv2, _, addr2 := types.KeyTestPubAddr()
-	acc1 := app.AccountKeeper.NewAccountWithAddress(ctx, addr1)
-	acc2 := app.AccountKeeper.NewAccountWithAddress(ctx, addr2)
-
-	acc2.SetPubKey(priv2.PubKey())
-
-	type args struct {
-		acc      exported.Account
-		sig      types.StdSignature
-		simulate bool
-	}
-	tests := []struct {
-		name    string
-		args    args
-		wantErr bool
-	}{
-		{"no sigs, simulate off", args{acc1, types.StdSignature{}, false}, true},
-		{"no sigs, simulate on", args{acc1, types.StdSignature{}, true}, false},
-		{"no sigs, account with pub, simulate on", args{acc2, types.StdSignature{}, true}, false},
-		{"pubkey doesn't match addr, simulate off", args{acc1, types.StdSignature{PubKey: priv2.PubKey()}, false}, true},
-		{"pubkey doesn't match addr, simulate on", args{acc1, types.StdSignature{PubKey: priv2.PubKey()}, true}, false},
-	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			_, err := ante.ProcessPubKey(tt.args.acc, tt.args.sig, tt.args.simulate)
-			require.Equal(t, tt.wantErr, !err.IsOK())
-		})
-	}
-}
-
-func TestConsumeSignatureVerificationGas(t *testing.T) {
-	params := types.DefaultParams()
-	msg := []byte{1, 2, 3, 4}
-
-	pkSet1, sigSet1 := generatePubKeysAndSignatures(5, msg, false)
-	multisigKey1 := multisig.NewPubKeyMultisigThreshold(2, pkSet1)
-	multisignature1 := multisig.NewMultisig(len(pkSet1))
-	expectedCost1 := expectedGasCostByKeys(pkSet1)
-	for i := 0; i < len(pkSet1); i++ {
-		multisignature1.AddSignatureFromPubKey(sigSet1[i], pkSet1[i], pkSet1)
-	}
-
-	type args struct {
-		meter  sdk.GasMeter
-		sig    []byte
-		pubkey crypto.PubKey
-		params types.Params
-	}
-	tests := []struct {
-		name        string
-		args        args
-		gasConsumed uint64
-		shouldErr   bool
-	}{
-		{"PubKeyEd25519", args{sdk.NewInfiniteGasMeter(), nil, ed25519.GenPrivKey().PubKey(), params}, types.DefaultSigVerifyCostED25519, true},
-		{"PubKeySecp256k1", args{sdk.NewInfiniteGasMeter(), nil, secp256k1.GenPrivKey().PubKey(), params}, types.DefaultSigVerifyCostSecp256k1, false},
-		{"Multisig", args{sdk.NewInfiniteGasMeter(), multisignature1.Marshal(), multisigKey1, params}, expectedCost1, false},
-		{"unknown key", args{sdk.NewInfiniteGasMeter(), nil, nil, params}, 0, true},
-	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			res := ante.DefaultSigVerificationGasConsumer(tt.args.meter, tt.args.sig, tt.args.pubkey, tt.args.params)
-
-			if tt.shouldErr {
-				require.False(t, res.IsOK())
-			} else {
-				require.True(t, res.IsOK())
-				require.Equal(t, tt.gasConsumed, tt.args.meter.GasConsumed(), fmt.Sprintf("%d != %d", tt.gasConsumed, tt.args.meter.GasConsumed()))
-			}
-		})
-	}
-}
+// func TestProcessPubKey(t *testing.T) {
```

Comment:
Why commented out?

Reply:
ProcessPubKey is no longer a function in `x/auth`